Libraries/Animated/src/NativeAnimatedHelper.js

```diff
@@ -74,13 +74,12 @@ var API = {
 var PROPS_WHITELIST = {
   style: {
     opacity: true,
-
+    transform: true,
     /* legacy android transform properties */
     scaleX: true,
     scaleY: true,
-    rotation: true,
     translateX: true,
-    translateY: true,
+    translateY: true
```

Comment:
the linter doesn't seem to be working on correct line numbers here, but you'll want to update the PR to include lingering commas for each element of lists of things (object notation, function calls, etc). This makes for cleaner diffs when people add/remove things.